tools/build-docs/build-docs

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+scriptDir=$(dirname $0)
+cflibDir=$scriptDir/../../cflib
+apiRefDir=$scriptDir/../../docs/api
+templateDir=$apiRefDir/template
+
+mkdir -p $apiRefDir
+mkdir -p $scriptDir/tmp
+
+[ -n "$API_REF_BASE" ] || {
+    export API_REF_BASE=$(readlink -fn $apiRefDir)
+}
+
+#
+# If cflib is not installed (mostly its dependencies) then pdoc3 cannot
+# traverse the library and get docstrings.
+#
+pip3 show cflib > /dev/null 2>&1 || {
+    pip3 install -e $scriptDir/../../
```

Comment:
A bit of a side effect if someone runs the script locally but makes sense in the web-builder container :-) I think this is good enough.